### PR TITLE
bar handles: add deactivate option

### PR DESCRIPTION
**Pull request by mehl321**
Create a new option to remove the handles in the slider bar so that the drag feature remains available even when having a very small date range. At the moment the handle bar are above so we can't do this.

### CODE REVIEW

**Comment by ghusse** [comment]
Is it possible to create unit tests for this option?

File: tests/unit/tests.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ var el = null;
 		},
 		function() {
 			// Default values tests
-			QUnit.deepEqualEpsilon(el.rangeSlider("option", "bounds"), { min:0, max:100 }, 1e-3, "Default bounds should be 0-100");
```
Comment (mehl321) [inline tests/unit/tests.js]: This line was duplicated with the one after.

**Comment by mehl321** [comment]
@ghusse What do you think about this test?

**Comment by ghusse** [comment]
Great.

Can you also create another PR on the gh-pages branch to update the docs?

**Comment by ghusse** [comment]
Can you also check that setting the option with the option method works (that needs to create handles/remove handles on the fly).

File: package.json [hunk 2]
```diff
@@ -2,6 +2,7 @@
   "name": "jQRangeSlider",
   "version": "5.7.1",
   "scripts": {
+    "grunt": "grunt",
```
Comment (mehl321) [inline package.json]: This allows to run grunt by doing `npm run grunt`, installed locally only.

**Comment by mehl321** [comment]
@ghusse I updated the code so that it's possible to update the option after the object construction. The code is not always self explanatory (there are no comments) so don't hesitate to comment now if something is not right.

**Pull request closed without merging**